gcloud/bigtable/happybase/test_batch.py

```diff
@@ -352,7 +486,7 @@ def test_it(self):
         expected_result = [
             ['cf1', None],
             ['cf2', None],
-            ['cf3', ''],
+            ['cf3', ''],  # AGGGHHHHHHHHHH THIS IS WEIRD
```

Comment:
Well that's the question, should it be valid. IMO it should not. The only valid identifiers should be `fam`, `fam:` and `fam:col`.

Though [HappyBase][1] uses `rstrip`.

[1]: https://github.com/wbolster/happybase/blob/9cbd718c10a3089f234f1eac1236b631e1f8e7cd/happybase/table.py#L60-L61